src/lib/jsonParse/jsonParse.cpp

```diff
@@ -445,5 +456,11 @@ std::string jsonParse
     }
   }
 
+  if (reqTimeStatistics)
```

Comment:
The one-liner alternative probably don't fit very well in this point... so, probably it is not a good idea at the end.